src/main/java/com/logpresso/scanner/Log4j2Scanner.java

```diff
@@ -400,15 +402,19 @@ private void fix() {
 				if (lockError) {
 					reportError(f, "Cannot lock file " + t.getMessage(), t);
 				} else if (truncateError) {
-					backupFile.delete();
+					if(!backupFile.delete()) {
+					  System.out.println("backup file cannot be deleted");
+ 					}
 					reportError(f, "Cannot truncate file " + t.getMessage(), t);
 				} else {
 					reportError(f, "Cannot backup file " + t.getMessage(), t);
 				}
 			} finally {
 				// restore read only attribute
 				if (readonlyFile)
-					f.setReadOnly();
+					if(!f.setReadOnly()) {
+                                          System.out.println("File cannot be set as Read Only");
```

Comment:
User should be notified which file was problem
`System.out.println("Error: File cannot be set as read only - " + targetFile.getAbsolutePath());`